core/assets/bundles/bundle_fr.properties

```
@@ -924,6 +924,8 @@ block.inverted-sorter.name = Trieur Inversé
 block.message.name = Message
 block.illuminator.name = Illuminateur
 block.illuminator.description = Une petite source lumineuse compacte et configurable. Nécessite de l'énergie pour fonctionner.
+block.large-illuminator.description = Grand Illuminateur
+block.large-illuminator.description = Un plus grande version de l'illuminateur. Il fournit une lumière configurable sur une surface large. Il requiert de l'énergie pour fonctionner.
```

Sorry for the mistake 😅
```suggestion
block.large-illuminator.description = Une plus grande version de l'illuminateur. Il fournit une lumière configurable sur une surface large. Il requiert de l'énergie pour fonctionner.
```